torch/csrc/jit/serialization/export.cpp

```diff
@@ -598,6 +609,37 @@ void EncoderBase::AddInitializersIntoGraphProto(
   }
 }
 
+size_t EncoderBase::GetGraphProtoSize(
+    onnx::GraphProto* graph_proto,
+    const std::shared_ptr<Graph>& graph,
+    const std::map<std::string, at::Tensor>& initializers) {
+  size_t sizes = 0;
+  for (auto input : graph->inputs()) {
+    auto name_tensor_pair = initializers.find(input->debugName());
+    if (name_tensor_pair == initializers.end()) {
+      continue;
+    }
+    onnx::GraphProto* graph_proto_copy = new onnx::GraphProto(*graph_proto);
```

Comment:
Do we delete this pointer at the end? or use `unique_ptr` instead?

Reply:
The main space occupied variable is `tensor_proto`. The other variables are released at the end of the function. So I think it's no need to delete it.

Reply:
Seems all [the function](https://github.com/pytorch/pytorch/blob/17df0052e08c9c16cbcafc080141d499bd3dd22b/torch/csrc/jit/serialization/export.cpp#L196) of `graph_proto` type is GraphProto pointer. Mixed `*pointers` and `shared_ptr` might not a good design. So I deleted the `graph_proto_copy` in the end of GetGraphProtoSize().